deploy/crds/shipwright.io_buildruns.yaml

```diff
@@ -42,14 +42,10 @@ spec:
         description: BuildRun is the Schema representing an instance of build execution
         properties:
           apiVersion:
-            description: 'APIVersion defines the versioned schema of this representation
-              of an object. Servers should convert recognized schemas to the latest
-              internal value, and may reject unrecognized values. More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#resources'
+            description: 'APIVersion defines the versioned schema of this representation of an object. Servers should convert recognized schemas to the latest internal value, and may reject unrecognized values. More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#resources'
```

Comment:
I think with the controller-gen update that I recently did, the generated yaml now had line breaks everywhere. May you check ?

Reply:
I've just re-run the controller-gen updates, and it modified those files. Let me know if that's fixed now.